Cargo.toml

```diff
@@ -38,6 +42,14 @@ homepage.workspace = true
 publish.workspace = true
 rust-version.workspace = true
 
+include = [
+    "bzip2.rs",
+    "bzip2recover.rs",
+    "Cargo.toml",
+    "./README.md",
+    "./LICENSE",
+]
```

Comment:
by default all files (also testing files) would be included, and the max package size is exceeded.